apps/theme/layouts/forms/DefaultFormLayout.tsx

```diff
@@ -40,7 +63,35 @@ const DefaultFormLayout: FormLayoutComponent = ({
     const [formSuccess, setFormSuccess] = useState(false);
 
     // All form fields - an array of rows where each row is an array that contain fields.
-    const fields = getFields();
+    const fields = getFields(currentStep);
+
+    // Check if the form is a multi step.
+    const isMultiStepForm = formData.steps.length > 1;
```

Comment:
Let's also have this passed down to us, with the next step / previous step callbacks. Thanks!